### PR TITLE
Add a simple console consumer tool

### PR DESCRIPTION
**Pull request by wvanbergen**
First simple version of a console consumer. Might be useful for testing purposes, and serves as a nice consumer example as well.

### CODE REVIEW

**Comment by wvanbergen** [comment]
@eapache

**Comment by wvanbergen** [comment]
I kind of like the idea of adding tools/ and examples/ subpackages to sarama itself.

File: tools/consoleconsumer/consoleconsumer.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		initialOffset = sarama.OffsetNewest
+	default:
+		initialOffset, offsetError = strconv.ParseInt(*offset, 10, 64)
+	}
+
+	if offsetError != nil {
+		logger.Fatalln("Invalid initial offset:", *offset)
+	}
+
+	config := sarama.NewConfig()
```
Comment (eapache) [inline tools/consoleconsumer/consoleconsumer.go]: you can still just pass `nil` for a default config

File: tools/consoleconsumer/consoleconsumer.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	c, err := sarama.NewConsumer(strings.Split(*brokerList, ","), config)
+	if err != nil {
+		logger.Fatalln(err)
+	}
+	defer c.Close()
```
Comment (eapache) [inline tools/consoleconsumer/consoleconsumer.go]: errcheck will complain about this :)

**Comment by eapache** [comment]
Dunno about examples (godoc includes examples already) but :+1: :+1:

**Comment by wvanbergen** [comment]
I find the example functions quite limited, because you are by definition limited to placing everything in a single function. This is great for small code snippets, but larger design ideas cannot really be expressed this way. E.g. remember the example I tried to write to demonstrate testing using a mock.

**Comment by eapache** [comment]
That's fair. Apart from the two minor nits this code LGTM, and I already used it for testing some Sarama changes :)

**Pull request merged**